### ISSUE TITLE
Colouring when working with multiple todo.txt files

### ISSUE THREAD

**Issue by dougwaterson** [mjdescy/TodoTxtMac#104]
Hi,
First of all, great work!  I just downloaded 2.1.1. (first time)
I like to keep a separate someday_maybe.txt as well as a todo.txt.  I set todo.txt as the default in preferences, and I opened someday_maybe.txt from finder->open with.
The someday_maybe.txt view has no colouring at all - just grey, whereas todo.txt has colouring.  I think the app supports multiple files...is the lack of colouring of the other file a bug?
Thanks
Doug

**Comment by mjdescy** [comment]
This sounds like a bug. I tried to reproduce it and could not. Is there any more information you could provide? Did you try to open the second file on the first run, or after changing the color settings and before closing the app? Does this behavior still occur after a restart of the app?

**Comment by dougwaterson** [comment]
My bad, it's not a bug.  Here's what I noticed:  
Make a new file, make one entry, take focus off the list (e.g. to the new task box), the one task in the list (which is highlighted) is all grey.  I guess I expected the colouring to take effect once I had moved focus to the new task box.
However, once more tasks are added I see it's only the last task that had focus which is all grey.
False alarm.
